container-storage-setup.sh

```diff
@@ -2072,13 +2115,11 @@ list_config() {
 
   echo "STORAGE_DRIVER=$_M_STORAGE_DRIVER"
 
-  if [ "$_M_STORAGE_DRIVER" == "" ]; then
-    return 0
-  elif [ "$_M_STORAGE_DRIVER" == "overlay" ]  || [ "$_M_STORAGE_DRIVER" == "overlay2" ];then
-    list_overlay_params
-  else
-    list_devicemapper_params
-  fi
+  case "$_M_STORAGE_DRIVER" in
+      ""|auto) return 0;;
```

Comment:
I don't think we can keep _M_STORAGE_DRIVER=auto. Because we use this value to determine how to activate storage. If we store "auto", then nothing will be activated and direct mode will be broken.

In direct mode we will have to translate it to either overlay2 or devicemapper and save that value.